### DISCUSSION TITLE
Need to set explicit Other SSH Client

### DISCUSSION THREAD

**Discussion by janhavel1** [gitextensions/gitextensions#10721] [Show and tell]
Hi all,

I would like to share my experience, when Git Extensions started to show up prompt to "enter passphrase for key ..." for pull and push.
Git operation from command line worked just fine, without any prompts and I was not aware of any change (except uninstall & upgrade to latest GE 4.0.2).

After few hours of investigations, I solved it by switching from default OpenSSH to Other ssh client 
![image](https://user-images.githubusercontent.com/92371511/218484640-0b941d51-62fc-47c9-a189-b09dada64ec4.png)

GIT_SSH=C:\Windows\System32\OpenSSH\ssh.exe

Hopefully it helps someone else.
I remember it happened once some time ago, so it may be a memo also for me.

If some one has an explanation...

Regards
Jan

**Comment by RussKie** [comment]
Thanks for sharing. I wouldn't be surprised if some discussions under https://github.com/gitextensions/gitextensions/issues?q=label%3A%22area%3A+ssh%2Fputty%22 contain an answer to your question.

**Comment by janhavel1** [comment]
Hi Igor,
funny, I found my issue from 2021 when it happened last time
https://github.com/gitextensions/gitextensions/issues/9646
So this new post is just a duplicate of it, I was not able to find it
yesterday.
Thank you for helping me :)
Rewards
Jan


út 14. 2. 2023 v 13:08 odesílatel Igor Velikorossov <
***@***.***> napsal:
